### ISSUE TITLE
Iron Key has empty value

### ISSUE THREAD

**Issue by AmProsius** [AmProsius/gothic-1-community-patch#177]
**Describe the bug**
The Iron Key has the label "Value" in the inventory, but the item has no value.

**Expected behavior**
The Iron Key no longer has a "Value" label.

**Additional context**
The key has a value of 0 and therefor the value isn't displayed. The German description of the key is *Eisenschlüssel*.

![ScreenShot_2021_3_5_20_40_24](https://user-images.githubusercontent.com/2817152/110169750-6d5be600-7df9-11eb-8b94-088b2aee71d7.jpg)

**Comment by AmProsius** [comment]
https://github.com/AmProsius/gothic-1-community-patch/blob/2b5e59952fbc8dacbade0a08317d9c519e5c6261/scriptbase/_work/Data/Scripts/Content/Items/MissionItems_1.d#L1145-L1156

changed to

```d
    TEXT[0]             = "Opens the door to the smithy's store.";
    ////COUNT[0]            = ;
    //TEXT[1]               = "";
    ////COUNT[1]            = ;
    //TEXT[2]               = "";
    //COUNT[2]          = ;
    //TEXT[3]           = "";
    //COUNT[3]          = ;
    //TEXT[4]               = "";
    ////COUNT[4]            = ;
    //TEXT[5]             = NAME_Value;
    //COUNT[5]            = value;
```

**Comment by szapp** [comment]
This bug will be resolved by #176.
